opengrok-indexer/src/test/java/org/opengrok/indexer/util/BooleanUtilTest.java

```diff
@@ -0,0 +1,96 @@
+package org.opengrok.indexer.util;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.opengrok.indexer.util.BooleanUtil; //IMPORT NOT USED
+
+import java.lang.reflect.Method; //IMPORT NOT USED
```

Comment:
Are these import not used because you removed the tests?

Reply:
Nope as no tests where removed from this PR